boilerplate/fullstack/remix/app/routes/supertokens.sessioninfo.$.tsx

```diff
@@ -0,0 +1,15 @@
+import { json } from "@remix-run/node";
+import { LoaderFunctionArgs } from "react-router-dom";
+import { getSessionDetails } from "../lib/superTokensHelpers.js";
+
+// Loader function for handling GET requests that also adds cache control headers
+export async function loader({ request }: LoaderFunctionArgs) {
+    try {
+        const res = await getSessionDetails(request);
+        return {
+            session: res.session,
```

Comment:
how are we able to send a session object in the result of this? Shouldn't this be serialisable?

Reply:
✔️